tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -89,13 +89,17 @@ public static async Task<OrderEntity> FetchOrderByOrderId(string connectionStrin
                           OrganisationId,
                           OrderStatusId,
                           Created,
+                          SupplierId,
+                          SupplierName,
+                          SupplierAddressId,
+                          SupplierContactId,
                           LastUpdated,
                           LastUpdatedBy,
                           LastUpdatedByName,
                           SupplierId,
                           SupplierName
                          FROM dbo.[Order]
-                         WHERE OrderId = @orderId;", new { orderId }));
+                         WHERE OrderId = @orderId;", new {orderId}));

```

Comment:
I prefer this with the white space, but if it ain't in the standards I can't complain!

Reply:
Nah I do too, dont know why it changed it